src/pages/Confirm/Confirm.styled.jsx

```diff
@@ -53,6 +53,13 @@ export const CheckImg = styled.img`
     width: 147px;
   }
 
+  @media (min-width: 1023px) {
+    margin-left: auto;
+    position: relative;
+    left: 5vw;
+    width: 100px;
+  }
+
   @media (min-width: 1200px) {
     margin-left: auto;
     position: relative;
```

Comment:
Here the image is positioned again using static values, this causes the image doesn't be responsive, please help us to improve this logic again.

I left you a similar comment in the media query 1025px block code

![imagen](https://user-images.githubusercontent.com/66505715/158850674-17f41ffc-ec1b-406f-8ac0-d4e2285ccfb8.png)